backend/Dockerfile

```
@@ -59,12 +59,18 @@ FROM base as development
 CMD /bin/sh -c "yarn install && yarn run dev"
 
 ##################################################################################
-# BUILD (Does contain all files and is therefore bloated) ########################
+# CODE (Does contain all code files and is pushed to DockerHub for rebranding) ###
 ##################################################################################
-FROM base as build
+FROM base as code
 
-# Copy everything
+# copy everything, but do not build.
 COPY . .
+
+##################################################################################
+# BUILD (Does contain all files and the compilate and is therefore bloated) ######
+##################################################################################
+FROM code as build
+
```

```suggestion
FROM code as build
```
This is quite self-explanatory.

---

We have this comments ahead of all states to show where a new state is starting. Therefore we leave it for kind of code design issues.